Base/etc/FileIconProvider.ini

```diff
@@ -10,7 +10,7 @@ javascript=*.js,*.mjs
 json=*.json
 library=*.so,*.a
 markdown=*.md
-music=*.midi
+music=*.midi,*.flac
```

Comment:
FLAC isn't strictly a music format, it's a generic audio format and in fact, works best with speech. Therefore, it should be a sound icon just as WAV is.